### PR TITLE
add templates for introduction and welcome pages

### PR DESCRIPTION
**Pull request by SoloSynth1**

### CODE REVIEW

**Comment by SoloSynth1** [comment]
Hi can anyone review this PR? Thanks!

File: docs/1-introduction/motivation.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+## Project Objectives
+
+By achieving the following objectives, the project aims to simplify GPU environment setup, enhance performance analysis, and contribute to more informed decision-making in selecting the optimal tools for machine learning workloads:
+
+### 1. Automated GPU Setup
+Develop an automated process using Ansible to set up GPU environments for machine learning tasks. This ensures consistent configurations and minimizes human error.
+
+### 2. Ease of Management
+Create an environment that's easy to manage and maintain. By using Conda, we establish isolated environments with dependencies, simplifying software management.
```
Comment (yamihe19) [inline docs/1-introduction/motivation.md:14]: Is it ok to use contraction "that's"?

**Review by yamihe19 — APPROVED** [review]

File: docs/1-introduction/motivation.md (hunk 1 above)
Comment (SoloSynth1) [reply docs/1-introduction/motivation.md:14]: I think you are right that we should not use contractions here, but again this is a template, which I will revise the wordings later

**Comment by yamihe19** [comment]
LGTM

**Comment by SoloSynth1** [comment]
Thanks @yamihe19, merging this to `main`.

**Pull request merged**